frontend-react/src/utils/UserUtils.ts

```diff
@@ -0,0 +1,20 @@
+import { OktaAuth } from "@okta/okta-auth-js";
+import { GLOBAL_STORAGE_KEYS } from "../components/GlobalContextProvider"
```

Comment:
I like directly including the keys and configs instead of passing them around!

Style suggestion: include trailing `;` for statements for consistency.

(In fact, we need to fix the `login.txs` so it directly includes the config data instead of loading it externally and passing it into the component.)

Reply:
I did have to change it to receive an `OktaAuth` param because what the line `const OKTA_AUTH = new OktaAuth(oktaAuthConfig);` did was just create a new auth on top of our current, thus, we had two and logged out of the one. However, getting our config from any component really only takes a quick `useOktaAuth()` hook call, so it's not _so_ bad. I'd still love to find a way to refactor this so we could just call `logout()`.